src/storage/src/compaction/task.rs

```diff
@@ -151,7 +151,10 @@ impl<S: LogStore> CompactionTask for CompactionTaskImpl<S> {
 
         let input_ids = compacted.iter().map(|f| f.file_id).collect::<Vec<_>>();
         let output_ids = output.iter().map(|f| f.file_id).collect::<Vec<_>>();
-        info!("Compacting SST files, input: {input_ids:?}, output: {output_ids:?}");
+        info!(
+            "Compacting SST files, input: {input_ids:?}, output: {output_ids:?}, window: {:?}",
+            self.compaction_time_window
```

Comment:
nitpick: Consider using the same style
```suggestion
            "Compacting SST files, input: {:?}, output: {:?}, window: {:?}",
            input_ids, output_ids, self.compaction_time_window
```